instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/activities/Constants.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.activities
+
+internal const val HUNDRED_PERCENT = 100f
+internal const val DATA_SIZE = 200L
```

Comment:
I think this property here is confusing, not clear what data size means and where should it be applicable just by looking on name. Maybe we need to inline it back? Same for the `REPEATED_TEST` probably.

Reply:
Agreed I'll inline it back